src/elexmodel/client.py

```diff
@@ -239,16 +248,21 @@ def get_estimates(
             aggregates, prediction_intervals, reporting_units, nonreporting_units, unexpected_units
         )
 
+        self.conformalization_data_unit_dict = {alpha: {} for alpha in prediction_intervals}
+        self.conformalization_data_agg_dict = {alpha: {} for alpha in prediction_intervals}
```

Comment:
these should be defined in the constructor. In that case, `get_conformalization_data_unit` and `get_conformalization_data_agg` would just return empty dictionaries before `get_estimates` is run

Reply:
added to constructor as None (is this what you meant?)

Reply:
yup!